src/System.Drawing.Primitives/tests/project.json

```diff
@@ -14,14 +16,35 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00807-03"
   },
   "frameworks": {
-    "netstandard1.3": {}
+    "netstandard1.3": {},
+    "netstandard1.7": {
+      "dependencies": {
+        "System.Runtime.Serialization.Formatters": "4.3.0-beta-24520-06"
+      }
+    }
   },
   "supports": {
     "coreFx.Test.netcore50": {},
     "coreFx.Test.netcoreapp1.0": {},
     "coreFx.Test.net46": {},
     "coreFx.Test.net461": {},
     "coreFx.Test.net462": {},
-    "coreFx.Test.net463": {}
+    "coreFx.Test.net463": {},
+    "coreFx.Test.netcoreapp1.1-ns17": {
```

Comment:
@joperezr do we still need this hack after the nuget update? Or is there still work to clean this up?